build-configs/configs/BuildConfigType.js

```diff
@@ -8,6 +8,34 @@ export type FeatureFlagsType = {|
   introSlides: boolean
 |}
 
+type AndroidGoogleServicesConfig = {|
+  google_app_id: string,
+  gcm_defaultSenderId: string,
+  default_web_client_id: string,
+  ga_trackingId: null | string,
+  firebase_database_url: string,
+  google_api_key: string,
+  google_crash_reporting_api_key: string,
+  project_id: string
+|}
+
+type AppleGoogleServicesConfig = {
```

Comment:
This is just for firebase, right?

Reply:
For Google Play Services yes

Reply:
As we use it just for firebase, I'd favor that naming. But I won't object if we keep this.